app/views/static_pages/room.html.erb

```diff
@@ -57,25 +55,26 @@ var comm = new Icecomm('9gv3ODZorBS2kVuNwnugBeV432PcNX3Qn0HmW33ofxK/u5Xolu');
 
     comm.on('disconnect', function(options) {
       document.getElementById(options.callerID).remove();
-
+      comm.close();
```

Comment:
Ensure that the local audio/vidio stream is stopped when the window is closed - closing the window fires icecomm's disconnect event.